### DISCUSSION TITLE
Plot.geo fill property with and without projection

### DISCUSSION THREAD

**Discussion by nshiab** [observablehq/plot#2128] [Q&A] [answered]
Hello!

All the code and screenshots come from this notebook: https://observablehq.com/d/e20aca7d9cd89a5d

If I plot a geojson of oceans without a projection, I can fill the oceans with a colour.

<img width="657" alt="Screenshot 2024-08-08 at 3 36 00 PM" src="https://github.com/user-attachments/assets/d851e5f8-3560-49c5-b3da-35c95a03a0b1">

But if I add a projection, then everything gets filled, not just the oceans. 🤔 

<img width="667" alt="Screenshot 2024-08-08 at 3 37 50 PM" src="https://github.com/user-attachments/assets/ee44c768-f23b-4e9d-ade5-60c69eb4f65a">

Maybe I'm not understanding something, but this feels like a bug...

Thank you for your time and your amazing work!

**Comment by mbostock** [comment, marked as answer]
This is usually because you have a winding issue with your geometry.  Try this:

https://observablehq.com/@fil/rewind

**Reply by nshiab** [reply]
Ah! You're right. It works. Thanks a lot!
